knotx-core/knotx-api/src/main/java/com/cognifide/knotx/api/JsonObjectRequest.java

```diff
@@ -43,4 +47,8 @@ protected MultiMap fromJsonArray(final JsonArray array) {
 
         return map;
     }
+
+    protected String multiMapToString(MultiMap map) {
+        return Joiner.on("\n").withKeyValueSeparator("=").join((Iterable<Map.Entry<String, String>>) map.getDelegate());
```

Comment:
There is  guava dependency in knotx-core/knotx-api/pom.xml. I will investigate why travis build is falling